### PR TITLE
removed update manager

### LINKED ISSUE #108: No dependency management found for the default environment
No dependency management found for the default environment. If you want
to keep your dependencies managed, please submit `Pipfile` or `requirements.in` or `requirements-dev.in` file.

To generate a `Pipfile`, use:
```
$ pip install --upgrade --user thamos
$ thamos discover --src-path ./
$ git add Pipfile
$ git commit -m 'Add Pipfile for dependency management'
```

Make sure your `Pipfile` or `requirements.in` or `requirements-dev.in` is placed in the root of your Git repository.

/kind feature
/priority important-soon

### PR DESCRIPTION
**Pull request by Gkrumbach07**
## Related Issues and Dependencies

fixes: https://github.com/thoth-station/search/issues/108

## This introduces a breaking change

- [ x] No

## This Pull Request implements

This repo does not use python so it has no deps to be updated

### CODE REVIEW

**Comment by codificat** [comment]
/hold
found a problem

File: .thoth.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 host: khemenu.thoth-station.ninja
 
 managers:
-  - name: update
```
Comment (codificat) [inline .thoth.yaml:4]: You probably want to remove the whole manager `configuration`, i.e. the following 2 lines as well

Comment (KPostOffice) [reply .thoth.yaml:4]: ^^^

**Review by KPostOffice — APPROVED** [review]
/lgtm

**Pull request merged**